src/Dialogs/BulkRenamer/RenamerDialog.vala

```diff
@@ -77,12 +77,12 @@ public class Files.RenamerDialog : Gtk.Dialog {
         actions.add_action_entries (ACTION_ENTRIES, this);
         insert_action_group ("renamer", actions);
 
-        deletable = true;
```

Comment:
Dialogs shouldn't have close buttons since that's an ambiguous action. We want to make sure users are making an explicit choice to cancel or rename when closing the dialog